backend/src/api/ApiHandler.py

```diff
@@ -0,0 +1,56 @@
+import array
+from typing import Tuple
+
+from backend.src.MapAPI import MapAPI
+
+from backend.src.api.nominatim import get_lat_long
+from backend.src.api.overpass import nearby_point
+from backend.src.api.osrm import determine_travel_time
+from backend.src.api.osrm import TravelOptions
+
+
+class ApiHandler(MapAPI):
+
+    def convert(loc):
+        if not isinstance(loc, Tuple):
+            # Assume it's an address and convert
+            loc = get_lat_long(loc)
+        return loc
+
+    def get_travel_time(self, loc_1, loc_2) -> float:
```

Comment:
We should specify in MapAPI now that loc_1 and loc_2 are addresses (or at least this is what it looks like we are assuming)

Reply:
They could be either since we convert them!

Reply:
It would then be best to specify that it could be either :)